frontend/src/components/common/checkbox/styles.module.scss

```
@@ -0,0 +1,33 @@
+.checkbox {
+  display: inline-block;
+  padding-left: 27px;
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 1.35;
+  user-select: none;
+  cursor: pointer;
+  color: var(--color-primary-dark);
+  input {
+    position: relative;
+    appearance: none;
+    &::before {
+      content: '';
+      position: absolute;
+      right: 7px;
+      top: -9px;
+      box-sizing: border-box;
+      width: 15px;
+      height: 15px;
+      padding-top: 1px;
+      text-align: center;
+      border: 1px solid var(--color-button-background);
+      border-radius: 2px;
+    }
+    &:checked {
+      &::before {
+        content: url('../../../assets/img/checked.svg');
```

```suggestion
        content: url('assets/img/checked.svg');
```
should also like this

---

it doesn't find such route

---

okay